plot_goes_x.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import argparse
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import tables
+from Chandra.Time import DateTime
+
+from Ska.Matplotlib import plot_cxctime
+
+parser = argparse.ArgumentParser(description='')
+parser.add_argument('--out', type=str,
+                    default='goes_x.png',
+                    help='Plot file name')
+parser.add_argument('--h5',
+                    default='GOES_X_16.h5',
+                    help='HDF5 file name')
+args = parser.parse_args()
+
+
+
+
+h5 = tables.open_file(args.h5, mode='r')
+table = h5.root.data[:]
+# Use just last 3 days if available
+table = table[table['time'] >= (DateTime() - 3).secs ]
+h5.close()
+
+
+plt.figure(1, figsize=(6, 4))
+for wave, color in zip(['long', 'short'], ['red', 'blue']):
+    vals = table[wave]
+    vals[vals <= 0] = 1e-10
```

Comment:
Use numpy clip e.g..  `vals = vals.clip(min=1e-10)`.  If one cared about memory then the `out=` option is available (`vals.clip(min=1e-10, out=vals)`, but not needed here.